lib/posix/pthread.c

```diff
@@ -17,6 +17,7 @@
 #include <zephyr/posix/pthread.h>
 #include <zephyr/sys/slist.h>
 #include <zephyr/sys/util.h>
+#include <zephyr/posix/unistd.h>
```

Comment:
minor nit: this could probably be inserted after `pthread.h` to be in lexographic order, but not blocking.

Reply:
Done